src/constants.ts

```diff
@@ -1,7 +1,5 @@
-// based off FlyByWire Simulations Discord Bot - https://github.com/flybywiresim/discord-bot
-
 export enum Colors {
-    HD_BLUE = '#0761e2',
```

Comment:
This one is optional however, if you wish to keep the hex format all you need to do is prefix ```0x``` before the hex. Example: ```0x0761e2```

Reply:
Perfect, thank you!